### PR TITLE
delete SSR

### PR DESCRIPTION
**Pull request by trusktr**
## **Summary**

Remove SSR. It was experimental and incomplete, and namely not working with plugins, and just generally not an ideal way to manage code (duplication of Renderer). Plus I have better ideas ([discord discussion](https://discord.com/channels/713647066802421792/774847028160626729/1122320166483656805)) on the how to leverage tools like Solid Start or Astro for SSR and SSG in ways that will be way better than what we had going on.

## **What kind of change does this PR introduce?**

Feature removal: delete SSR.

## **For any code change,**

- [x] Related documentation has been updated if needed
- [x] Related tests have been updated or tests have been added

## **Does this PR introduce a breaking change?** (check one)

- [x] Yes
- [ ] No

Breaking change because someone might be using it. This requires a major version bump.

## **Related issue, if any:**

I am planning to finish a few more things in a row here:

https://github.com/docsifyjs/docsify/issues/2104.

We can wait for all of those (except perhaps TypeScript) before releasing a major version bump so that we don't release several major version bumps back to back.

## **Tested in the following browsers:**

- [x] Chrome
- [x] Firefox
- [x] Safari
- [x] Edge
- [ ] IE

### CODE REVIEW

File: test/integration/__snapshots__/docs.test.js.snap [hunk 1]
```diff
@@ -9,4 +9,4 @@ exports[`Docs Site coverpage renders and is unchanged 1`] = `
...
.../li></ul></li><li><p>Customization</p><ul><li><a href=\\"#/configuration\\" title=\\"undefined\\">Configuration</a></li><li><a href=\\"#/themes\\" title=\\"undefined\\">Themes</a></li><li><a href=\\"#/plugins\\" title=\\"undefined\\">List of Plugins</a></li><li><a href=\\"#/write-a-plugin\\" title=\\"undefined\\">Write a Plugin</a></li><li><a href=\\"#/markdown\\" title=\\"undefined\\">Markdown configuration</a></li><li><a href=\\"#/language-highlight\\" title=\\"undefined\\">Language highlighting</a></li><li><a href=\\"#/emoji\\" title=\\"undefined\\">Emoji</a></li></ul></li><li><p>Guide</p><ul><li><a href=\\"#/deploy\\" title=\\"undefined\\">Deploy</a></li><li><a href=\\"#/helpers\\" title=\\"undefined\\">Helpers</a></li><li><a href=\\"#/vue\\" title=\\"undefined\\">Vue compatibility</a></li><li><a href=\\"#/cdn\\" title=\\"undefined\\">CDN</a></li><li><a href=\\"#/pwa\\" title=\\"undefined\\">Offline Mode (PWA)</a></li><li><a href=\\"#/embed-files\\" title=\\"undefined\\">Embed Files</a></li></ul></li><li><p><a href=\\"#/awesome\\" title=\\"undefined\\">Awesome docsify</a></p></li><li><p><a href=\\"#/changelog\\" title=\\"undefined\\">Changelog</a></p></li></ul></div></aside>"`;
```
Comment (trusktr) [inline test/integration/__snapshots__/docs.test.js.snap:12]: It is very difficult to review these sorts of text-based snapshots. The only thing that changed here is that the server-side rendering sidebar menu item has been removed.

**Comment by sy-records** [comment]
Use Angular commit message to generate change logs

File: package.json [hunk 2]
```diff
@@ -98,7 +92,6 @@
     "eslint-plugin-prettier": "^4.0.0",
     "husky": "^8.0.3",
     "jest": "^27.4.7",
-    "lerna": "^7.0.2",
```
Comment (sy-records) [inline package.json:101]: why delete `lerna`?

Comment (trusktr) [reply package.json:101]: After deleting the docsify-server-render package, we don't need to manage a sub-package anymore.

The funny thing is that, I realized after we merged https://github.com/docsifyjs/docsify/pull/2094 that once we removed `lerna bootstrap`, we are not using `lerna` for anything, so we could simply deleted it. 😄

Comment (trusktr) [reply package.json:101]: I.e., now the only thing we need to do during development is `npm install` and `npm run whatever`.

**Comment by trusktr** [comment]
> Use Angular commit message to generate change logs

Updated

**Review by Koooooo-7 — APPROVED** [review]

**Comment by trusktr** [comment]
Looks like the codesandbox build sometimes fails when its system limits the number of process (EAGAIN error). 

Is there a way to make it retry?

**Comment by sy-records** [comment]
I tried to access the admin codesandbox before, but I don't have access.
Previously added by @anikethsaha , Can you help to see it?

**Comment by trusktr** [comment]
No reply from @anikethsaha yet.

@sy-records @Koooooo-7 codesandbox build is not required, so we can continue anyway.

**Review by sy-records — APPROVED** [review]

**Comment by trusktr** [comment]
@Koooooo-7 Because you already approved, and I merged develop and tests passed but it undid your approval, and then @sy-records approved, I'll go ahead and merge.

**Pull request merged**